src/site/filters/liquid.js

```diff
@@ -8,10 +8,7 @@ function getPath(obj) {
 }
 
 module.exports = function registerFilters() {
-  const {
-    featureFlags,
-    enabledFeatureFlags,
-  } = require('../utilities/featureFlags');
+  const { cmsFeatureFlags } = global;
```

Comment:
These are coming from `global` now because we shouldn't be `requiring` the hard-coded files. They get set [in `options.js#setUpFeatureFlags`](https://github.com/department-of-veterans-affairs/vets-website/pull/10586/files#diff-b18d5a20aff9790323e66b853e6c290fR206).